teku/src/main/java/tech/pegasys/teku/cli/options/DepositOptions.java

```diff
@@ -39,9 +50,13 @@ public int getEth1LogsMaxBlockRange() {
   }
 
   public void configure(final TekuConfiguration.Builder builder) {
+    ArrayList<String> mergedEth1Endpoints = new ArrayList<>();
```

Comment:
nit: Use the generic interface as the variable type.
```suggestion
    List<String> mergedEth1Endpoints = new ArrayList<>();
```